editor/editor_help.cpp

```diff
@@ -1379,7 +1379,17 @@ void EditorHelp::_update_doc() {
 		}
 
 		// Enums
+		bool has_enums = false;
 		if (enums.size()) {
+			for (KeyValue<String, DocData::EnumDoc> &E : cd.enums) {
+				if (cd.is_script_doc && cd.is_script_doc && E.key.begins_with("_") && E.value.description.strip_edges().is_empty()) {
```

Comment:
```suggestion
		bool has_enums = enums.size() && !cd.is_script_doc;
		if (enums.size() && !has_enums) {
			for (KeyValue<String, DocData::EnumDoc> &E : cd.enums) {
				if (E.key.begins_with("_") && E.value.description.strip_edges().is_empty()) {
```
No need to check these every time, the loop accepts enums if they're non-script doc ones, so we can skip this part

Might be a more elegant way to do this just a suggestion, but the loop is unnecessary if it's not a script doc

Reply:
yeah you're right.